index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Portfolio - Mobile Version</title>
+    </head>
+    <body>
+        <main>
+            <div class="header-section">
+                <header class="header">
+                    <nav class="nav">
+                        <img src="assets/my-logo.png" alt="My Logo" />
+                        <div class="hamburger-menu">
+                            <div class="top-line"></div>
+                            <div class="middle-line"></div>
+                            <div class="bottom-line"></div>
+                        </div>
+                    </nav>
+                </header>
+                <div class="profile-section">
+                    <p class="about-header">
+                        I’m Olawale Glad to see you!
+                    </p>
```

Comment:
- [ ] Since this is an important part of this page, it would be nice if you used the `h1` tag. The `h1` tag defines the most important heading of a page and I believe this part of your page is the most important.